components/components/navigation/menu-main/menu-main.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file
+ * This file is used to add any javascrip that is needed for the main menu.
+ */
+
+(function ($, Drupal) {
+  'use strict';
+  Drupal.behaviors.menutoggle = {
+    attach: function (context, settings) {
+      var toggler = $('.menu-main__toggle');
```

Comment:
We should use data attributes as selectors in JS

Reply:
@lauriii Why? Is there a coding standard about it? Can you point me to that?

Reply:
Found two issues:
- https://www.drupal.org/node/1090592
- https://www.drupal.org/node/2346799

Reply:
It is mainly to be able to have documentation that JavaScript expects some element to exist. If you use the classes used for styling the page, the person styling the page assumes they can safely change them to be whatever they would like.

Reply:
@lauriii Thanks for letting me know about this! :)